packages/sources/css/src/components/text-input/src/index.css

```diff
@@ -67,6 +61,24 @@ textarea.vtmn-text-input {
   line-height: 1;
 }
 
+label + div {
```

Comment:
It could be better to avoid this div and try another technique than `display: flex`

Reply:
Ok for the classname, need to investigate this part. Hard to position the icon manually without adding a div.